demos/interpreter/step-execution.html

```diff
@@ -82,8 +82,11 @@ <h1><a href="https://developers.google.com/blockly/">Blockly</a> &gt;
         </value>
       </block>
     </category>
-    <category name="Variables" custom="VARIABLE"></category>
-    <category name="Functions" custom="PROCEDURE"></category>
+    <sep></sep>
+    <category name="Variables" custom="VARIABLE" colour="%{BKY_VARIABLES_HUE}">
```

Comment:
Is the color attribute parsed when doing a custom category?  Should it be?

Reply:
This works.  The `colour` attribute is part of the total XML, not the XML generated by the custom category function.

I haven't done any test where a custom category function might return a `colour` attribute on its top-level XML element.  I'm guessing such an attribute will be ignored and no toolbox category label color will update.